src/common/meta/src/ddl/drop_flow.rs

```diff
@@ -102,18 +104,9 @@ impl DropFlowProcedure {
         let flownode_ids = &self.data.flow_info_value.as_ref().unwrap().flownode_ids;
         let flow_id = self.data.task.flow_id;
         let mut drop_flow_tasks = Vec::with_capacity(flownode_ids.len());
-        let cluster_id = self.data.cluster_id;
-
-        for flownode in flownode_ids.values() {
-            let peer = self
-                .context
-                .peer_lookup_service
-                .flownode(cluster_id, *flownode)
-                .await?
-                .with_context(|| UnexpectedSnafu {
-                    err_msg: "Attempted to drop flow on a node that could not be found. Consider verifying node availability.",
-                })?;
-            let requester = self.context.node_manager.flownode(&peer).await;
+
+        for FlowRouteValue { peer } in &self.data.flow_route_values {
```

Comment:
Why is the `partition_id` not in `DropRequest`?

Reply:
Good point🥹 I'm going to create an issue to fix this.

Reply:
OK